feedback2go/Views/CorrectionSidebar/GeneralFeedbackCellView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  GeneralFeedbackView.swift
+//  feedback2go
+//
+//  Created by Florian Huber on 13.11.22.
+//
+
+import SwiftUI
+
+struct GeneralFeedbackCellView: View {
```

Comment:
design needs some design changes. More particular the feedback area (delete, add and textfield). Discuss on Wednesday.

Reply:
It has the possiblity to add, delete and so on. Maybe you just couldn't see because of the fixed frame size.
Can you specify what you don't like?

Reply:
I think this discussion involves the other team members as well and we should resolve this conversation for now.